chia/consensus/blockchain.py

```diff
@@ -202,53 +215,12 @@ async def receive_block(
         if not genesis and (self.block_record(block.prev_header_hash).height + 1) != block.height:
             return ReceiveBlockResult.INVALID_BLOCK, Err.INVALID_HEIGHT, None, ([], {})
 
-        npc_result: Optional[NPCResult] = None
```

Comment:
All of this code was only being used for tests, since pre_validation_result is never None in full node.